src/js/views/conversationView.js

```diff
@@ -18,8 +21,32 @@ module.exports = Marionette.CompositeView.extend({
         messages: '[data-ui-messages]'
     },
 
-    scrollToBottom: function() {
-        this.$el.scrollTop(this.$el.get(0).scrollHeight - this.$el.outerHeight() - this.ui.logo.outerHeight());
+    initialize: function() {
+        // http://stackoverflow.com/questions/11600040/jquery-js-html5-change-page-content-when-keyboard-is-visible-on-mobile-devices
+
+        initialScreenSize = window.innerHeight;
+
+        /* Android */
+        window.addEventListener('resize', function() {
+            isKeyboard = (window.innerHeight < initialScreenSize);
+            isLandscape = (screen.height < screen.width);
+
+            this.keyboardToggled();
+        }.bind(this), false);
+
+        /* iOS */
+        $('.message-input').bind('focus blur', function() {
```

Comment:
use `this.$('.message-input')` to scope it to the widget. it's still is to broad and will target all `.message-input` in the page if any is present